docs/gh-actions/gh-update.md

```diff
@@ -15,44 +15,50 @@ This is only available with Loop 3 and Loop development branch.
     * 5 minutes to regenerate token
     * 5 minutes to update the GH_PAT Secret for each of your app repositories
 
-!!! abstract "Summary"
+??? abstract "`Page Summary` (click to open/close)"
```

Comment:
Since we want to translate `Page Summary`, it seems to me that backticks are not necessary here.
If the intention was to make the title stand out, we should find another trick to do that.